api/accessrequest/access_request.go

```diff
@@ -60,7 +60,11 @@ func GetResourceDetails(ctx context.Context, clusterName string, lister client.L
 		// We're interested in hostname or friendly name details. These apply to
 		// nodes, app servers, and user groups.
 		switch resourceID.Kind {
-		case types.KindNode, types.KindApp, types.KindUserGroup, types.KindIdentityCenterAccount:
+		case types.KindIdentityCenterAccount:
+			resourceID.Kind = types.KindApp
+			resourceIDs = append(resourceIDs, resourceID)
```

Comment:
Does the GetResourceDetails->GetResourcesByResourceIDs->GetResourcesByKind->[mapResourceKindToListResourcesType](https://github.com/gravitational/teleport/blob/e1be633ac0cdb49614954bea03ef4f505736d5d3/api/accessrequest/access_request.go#L148) is not better way to overwrite the kind ?  KindIdentityCenterAccount-> types.KindApp ?